### ISSUE TITLE
Release tests v5.2.0 🏁

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/parabol#3709]
- Add the `release` label to this issue
- The basic checklist is required
- Adding tests from merged PRs is optional
## The basics
Run through this list at least once:
- [ ] Smoke tested the Demo, unauthenticated
- [ ] Created an account
- [ ] Verified invite via mass link works
- [ ] Verified invite via email works
- [ ] Added Slack, verified meeting notification
- [ ] Added GitHub, verified issue created
- [ ] Added Jira, verified issue created
- [ ] Smoke tested the Retro meeting with 2 players
- [ ] Smoke tested the Action meeting with 2 players
- [ ] Smoke tested cards on the dashboard
- [ ] Created a 2nd team
- [ ] Created a 2nd organization
- [ ] Upgraded to Pro
- [ ] Smoke tested the app on a mobile device (e.g. navigate between views, smoke test a Retro meeting, etc.)
## What’s changed
At your discretion, complete the tests for any merged PRs:
- List each PR with a header and link to ([title], #[issue id])
- Copy and paste tests, or note what was tested, in this issue
- Run the equivalent test on staging that you would run in a local environment as closely as possible (e.g. updating the database to simulate cases, etc.)

**Comment by ackernaut** [comment]
### Timer display doesn’t appear in demo
- Start the demo, try the timer, it doesn’t appear

**Comment by ackernaut** [comment]
### Last topic highlighted like the facilitator’s location
- Going back into a completed meeting, the last topic (stage) that was completed has the facilitator’s location highlight
- It doesn’t matter which user you are when going back to the completed meeting

![Screenshot 2020-03-26 09 54 18](https://user-images.githubusercontent.com/307286/77660921-ef842f00-6f47-11ea-9ca3-bbd8637661cc.png)

**Comment by mattkrick** [comment]
yep! feature not a bug :yum: i removed the "join facilitator" button & disabled drag & drop on discussion items, & made it so the facilitator stage doesn't update when you navigate post-end. i figure the pink is good to show where the meeting ended

**Comment by ackernaut** [comment]
I like the idea, I wonder if we put a little `flag` icon and maybe a different color, but future tension

**Comment by ackernaut** [comment]
Okay, a few things to note:
- We can hotfix the timer display. It’s not showing in the demo or real retro.
- The last topic viewed is highlighted like the facilitator location. We can consider UI design improvements later.
- I’ve noticed a few times now (just didn’t write it down) that in the demo summary all the users are marked as absent. I think they should be marked as present since they were in the top right and doing things in the demo.
- I’ve got other UI polish things I’m noting. Will probably push a minimal PR sooner that later.
